ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,61 +13,75 @@
-    Select a KV engine mount and path to sync a secret to the destination.
-  </p>
-
+<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
   <MessageError @errorMessage={{this.error}} />
 
+  {{#if this.syncedSecret}}
+    <Hds::Alert @type="inline" @color="success" as |A|>
+      <A.Title>Successfully synced a secret</A.Title>
+      <A.Description data-test-sync-success-message>
+        Sync operation successfully initiated for "{{this.syncedSecret}}". You can continue on this page to sync more
+        secrets.
+      </A.Description>
+    </Hds::Alert>
+  {{/if}}
+
   <div class="has-top-margin-l">
-    {{#if this.mounts}}
-      <SearchSelect
-        @label="Select a mount for the KV engine"
-        @options={{this.mounts}}
-        @selectLimit={{1}}
-        @disallowNewItems={{true}}
-        @onChange={{this.setMount}}
-        data-test-sync-mount-select
-      />
-    {{else}}
-      <FormFieldLabel @label="Enter an existing KV engine mount" />
-      <FilterInput
-        placeholder="KV engine mount path"
-        value={{this.mountPath}}
-        @onInput={{fn (mut this.mountPath)}}
-        data-test-sync-mount-input
-      />
-    {{/if}}
 
-    <KvSuggestionInput
-      @label="Select a secret to sync"
-      @subText="Enter the full path to the secret. Suggestions will display below if permitted by policy."
-      @value={{this.secretPath}}
-      @mountPath={{this.mountPath}}
-      @onChange={{fn (mut this.secretPath)}}
-    />
+    <p class="is-label">Which secrets would you like us to sync?</p>
+    <p class="sub-text">
+      Select a KV engine mount and path to sync a secret to the destination.
+    </p>
 
-    <div class="field box is-fullwidth is-bottomless has-top-margin-xxxl">
-      <Hds::ButtonSet>
-        <Hds::Button
-          @text="Sync to destination"
-          @color="primary"
-          @icon={{if this.setAssociation.isRunning "loading"}}
-          type="submit"
-          disabled={{or (not (and this.mountPath this.secretPath)) this.isSecretDirectory this.setAssociation.isRunning}}
-          data-test-sync-submit
+    <div class="has-top-margin-l">
+      {{#if this.mounts}}
+        <SearchSelect
+          @label="Select a mount for the KV engine"
+          @options={{this.mounts}}
+          @selectLimit={{1}}
+          @disallowNewItems={{true}}
+          @onChange={{this.setMount}}
+          data-test-sync-mount-select
         />
-        <Hds::Button @text="Back" @color="secondary" {{on "click" this.back}} data-test-sync-cancel />
-      </Hds::ButtonSet>
-      {{#if this.isSecretDirectory}}
-        <AlertInline
-          @type="warning"
-          @message="Syncing secret directories is not available at this time, please type a path to a single secret"
-          class="has-top-margin-s"
+      {{else}}
+        <FormFieldLabel @label="Enter an existing KV engine mount" />
+        <FilterInput
```

Comment:
Ideally we need `FilterInput` to get an id, and then we can add `for="input-id"` to the FormFieldLabel. I can do that as a follow-on if you don't want to handle that here, just let me know.

Reply:
Sure I will update that.